apps/codelab/src/app/components/exercise/exercise.component.html

```diff
@@ -24,6 +25,10 @@
         [bootstrap]="bootstrapTest"
         [code]="code"
       ></codelab-babel-test-runner>
+      <div *ngIf="isSolved" class="solved-message">
+        <h3>Go to the next slide</h3>
```

Comment:
please add i18n attribute so this can be translated

Reply:
Exactly. I'll do it asap.